Doppler.AccountPlans/Infrastructure/AccountPlansRepository.cs

```diff
@@ -29,5 +33,14 @@ public async Task<IEnumerable<PlanDiscountInformation>> GetPlanDiscountInformati
 
             return result;
         }
+
+        public async Task<PlanAmountDetails> GetAdjustmentPlan(int planId, string accountName)
```

Comment:
Perhaps the name of the method can be: ```getPlanAmountDetails```. Do you think?

Reply:
Yeah I'll change it